src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/resources/viewmodel/ResourceListViewModel.kt

```diff
@@ -6,16 +6,30 @@ import org.wycliffeassociates.otter.jvm.app.ui.resourcetakes.viewmodel.RecordRes
 import org.wycliffeassociates.otter.jvm.app.ui.workbook.viewmodel.WorkbookViewModel
 import org.wycliffeassociates.otter.jvm.app.widgets.resourcecard.model.ResourceGroupCardItemList
 import org.wycliffeassociates.otter.jvm.app.widgets.resourcecard.model.resourceGroupCardItem
+import org.wycliffeassociates.otter.jvm.utils.onChangeAndDoNow
 import tornadofx.*
 
-class ResourcesViewModel : ViewModel() {
+class ResourceListViewModel : ViewModel() {
     internal val recordResourceViewModel: RecordResourceViewModel by inject()
     private val workbookViewModel: WorkbookViewModel by inject()
 
     val resourceGroupCardItemList: ResourceGroupCardItemList = ResourceGroupCardItemList()
 
-    fun loadResourceGroups() {
-        val chapter = workbookViewModel.chapter
+    init {
+        workbookViewModel.activeChapterProperty.onChangeAndDoNow {
+            it?.let { targetChapter ->
+                loadResourceGroups(getSourceChapter(targetChapter))
+            }
+        }
+    }
+
+    private fun getSourceChapter(targetChapter: Chapter): Chapter {
```

Comment:
As of now, activeChapter and activeChunk in WorkbookViewModel correspond to the `target` Book in the workbook. But the resources are found in the `source` book. Not sure if we want to handle this differently.

Reply:
That seems correct to me.